app/models/course/lesson_plan/milestone.rb

```diff
@@ -1,9 +1,15 @@
 # frozen_string_literal: true
 class Course::LessonPlan::Milestone < ApplicationRecord
-  belongs_to :course, inverse_of: :lesson_plan_milestones
+  acts_as_lesson_plan_item(has_todo: false)
```

Comment:
Sorry, small syntax nitpick, there shouldn't be parentheses here, as with all the other `acts_as` lines.